[2021-11-17T19:38:06.114692+00:00] php.DEBUG: Warning: filemtime(): stat failed for /home/csattoszsolt/MyProjects/Trendency/local/skeleton/ecosystem/application/backend/config/routes/dev/web_profiler.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/app/vendor/symfony/config/Resource/FileResource.php","line":62,"trace":[{"file":"/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-11-17T19:38:06.164117+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-17T19:38:06.319397+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/favicon.ico" (from "http://localhost:8000/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:06.341415+00:00] php.DEBUG: Warning: filemtime(): stat failed for /home/csattoszsolt/MyProjects/Trendency/local/skeleton/ecosystem/application/backend/config/routes/dev/web_profiler.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/app/vendor/symfony/config/Resource/FileResource.php","line":62,"trace":[{"file":"/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-11-17T19:38:06.392762+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:38:06.393209+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:06.394490+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:06.521203+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:38:06.521335+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:09.163774+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-17T19:38:09.306278+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/favicon.ico" (from "http://localhost:8000/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:09.355263+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:38:09.355446+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:09.356020+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:09.488329+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:38:09.488481+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:15.881449+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/favicon.ico" (from "http://localhost:8000/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:15.923794+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:38:15.923998+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:15.924640+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:38:16.050421+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:38:16.050555+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:46.478354+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-18T13:59:46.678301+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/favicon.ico" (from "http://localhost:8000/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:46.734695+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:46.734945+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:46.735755+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:46.862038+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:46.862199+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/favicon.ico\" (from \"http://localhost:8000/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T13:52:12.354304+00:00] console.CRITICAL: Error thrown while running command "cache:clear -y". Message: "The "-y" option does not exist." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\RuntimeException(code: 0): The \"-y\" option does not exist. at /app/vendor/symfony/console/Input/ArgvInput.php:198)","command":"cache:clear -y","message":"The \"-y\" option does not exist."} []
[2021-11-19T13:52:12.355340+00:00] console.DEBUG: Command "cache:clear -y" exited with code "1" {"command":"cache:clear -y","code":1} []
[2021-11-19T17:04:21.117121+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T17:04:21.497087+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T17:04:21.617838+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T17:04:21.618344+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T17:04:21.621562+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T17:04:21.850391+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T17:04:21.850643+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T17:04:23.837041+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T17:04:24.081992+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8081/favicon.ico" (from "http://localhost:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T17:04:24.166693+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T17:04:24.167256+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T17:04:24.169032+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T17:04:24.399039+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T17:04:24.399287+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:25.752210+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:07:25.995430+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8081/favicon.ico" (from "http://localhost:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:26.046371+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:07:26.046559+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:26.047394+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:26.171048+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:07:26.171190+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8081/favicon.ico\" (from \"http://localhost:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:38.163108+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:07:38.305584+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:38.351297+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:07:38.351581+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:38.352385+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:07:38.476061+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:07:38.476198+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:29.203956+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:28:29.410878+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.skeleton.local:8081/favicon.ico" (from "http://api.application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.skeleton.local:8081/favicon.ico\" (from \"http://api.application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:29.460216+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:28:29.460476+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.skeleton.local:8081/favicon.ico\" (from \"http://api.application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:29.461157+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.skeleton.local:8081/favicon.ico\" (from \"http://api.application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:29.586159+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:28:29.586290+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.skeleton.local:8081/favicon.ico\" (from \"http://api.application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:50.115090+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:28:50.266534+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/favicon.ico" (from "http://application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:50.314194+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:28:50.314446+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:50.315175+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:50.444686+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:28:50.444815+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:53.799680+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:28:53.945093+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/favicon.ico" (from "http://application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:53.991416+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:28:53.991676+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:53.992333+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:54.133977+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:28:54.134111+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:59.788491+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:28:59.935302+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/favicon.ico" (from "http://application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:59.981869+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:28:59.982305+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:28:59.983113+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:00.113956+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:00.114102+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:00.646457+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:29:00.780723+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/favicon.ico" (from "http://application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:00.833370+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:00.833638+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:00.834594+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:00.959161+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:00.959295+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.127411+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:29:01.263893+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/favicon.ico" (from "http://application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.310060+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:01.310252+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.310938+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.314696+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:29:01.444631+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:01.444758+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.450920+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/favicon.ico" (from "http://application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.497560+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:01.497917+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.498780+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.629180+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:01.629315+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.748575+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:29:01.894475+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://application.skeleton.local:8081/favicon.ico" (from "http://application.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.941963+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:01.942144+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:01.942717+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:02.075546+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:02.075757+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://application.skeleton.local:8081/favicon.ico\" (from \"http://application.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:35.943085+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.skeleton.local:8081/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8081/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:29:36.133108+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.skeleton.local:8081/favicon.ico" (from "http://api.cms.skeleton.local:8081/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8081/favicon.ico\" (from \"http://api.cms.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:36.179441+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:36.179741+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8081/favicon.ico\" (from \"http://api.cms.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:36.180703+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8081/favicon.ico\" (from \"http://api.cms.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:36.307693+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:36.307886+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8081/favicon.ico\" (from \"http://api.cms.skeleton.local:8081/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:40.220833+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.skeleton.local:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:29:40.368222+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.skeleton.local:8080/favicon.ico" (from "http://api.cms.skeleton.local:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:40.413344+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:40.413628+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:40.414208+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:40.540120+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:40.540258+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:42.380110+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.skeleton.local:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-19T19:29:42.527912+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.skeleton.local:8080/favicon.ico" (from "http://api.cms.skeleton.local:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:42.574021+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:42.574198+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:42.574755+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-19T19:29:42.706605+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-19T19:29:42.706731+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.skeleton.local:8080/favicon.ico\" (from \"http://api.cms.skeleton.local:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:27:56.330964+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:34:34.836124+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:34:37.923715+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:34:38.289735+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:34:38.473967+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:36:31.012486+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://172.22.0.3/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:36:31.180464+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://172.22.0.3/favicon.ico" (from "http://172.22.0.3/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:36:31.277638+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:36:31.278117+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:36:31.279629+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:36:31.404847+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:36:31.404985+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:36:50.467229+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://172.22.0.3/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:36:50.664391+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://172.22.0.3/favicon.ico" (from "http://172.22.0.3/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:36:50.711541+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:36:50.711802+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:36:50.712412+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:36:50.838857+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:36:50.839044+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://172.22.0.3/favicon.ico\" (from \"http://172.22.0.3/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:37:31.706759+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:37:31.864753+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/favicon.ico" (from "http://api.application.localhost/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:37:31.909340+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:37:31.909531+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:37:31.910554+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:37:32.042491+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:37:32.042621+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:58:09.346843+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T11:58:09.991210+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/favicon.ico" (from "http://api.application.localhost/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:58:10.080695+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:58:10.081266+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:58:10.082447+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T11:58:10.292458+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T11:58:10.292651+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/favicon.ico\" (from \"http://api.application.localhost/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T12:26:29.147931+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T12:28:26.123717+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T12:44:07.274835+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T13:17:12.362556+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.application.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.application.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
